### ISSUE TITLE
Script error at manager-base.js:82

### ISSUE THREAD

**Issue by cornhundred** [ipython/ipywidgets#1127]
Hi, I am getting a new error for a widget I built using the [cookie cutter template](https://github.com/jupyter-widgets/widget-cookiecutter)

``Uncaught Error: Script Error for "clustergrammer_widget" `` 

in the file 

``manager-base.js:82``

in the developer console. I am getting the error on previous commits that used to work. 

I ran ``pip install -e .`` to install the local module. The repo is [clustergrammer_widget](https://github.com/MaayanLab/clustergrammer-widget)

**Comment by cornhundred** [comment]
I'm also getting the following error: 

```
jupyter nbextension enable --py --sys-prefix clustergrammer_widget
Enabling notebook extension clustergrammer_widget/extension...
      - Validating: problems found:
        - require?  X clustergrammer_widget/extension
```

**Comment by cornhundred** [comment]
I noticed that this command 

``jupyter nbextension install --py --symlink --user clustergrammer_widget``

was giving an error (#17) about the file already existing, so I tried deleting the file by clearing out the 

``/Users/my-name/Library/Jupyter/nbextensions``

directory. After doing this I was able to get the first command to work as well as the 

``jupyter nbextension enable --py --sys-prefix clustergrammer_widget``

command. Now I am able to get my local widget to work correctly.
